.github/workflows/docker-build-and-run.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Run Docker
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-run-docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      # Build the Docker image
+      - name: Build Docker image
+        run: |
+          docker build -t tf2.4_ivim-mri_codecollection -f Docker/Dockerfile .
```

Comment:
Is this docker image getting saved as an artifact? I'm not sure how that works in github actions but it's probably possible. At the very least I know it's possible to dump docker images to files.

Reply:
I save the image as an artifact now.